deployment/packer/template.js

```diff
@@ -113,7 +114,7 @@
                 "sudo apt-get install python-pip python-dev -y",
                 "sudo pip install paramiko==1.16.0",
                 "sudo pip install ansible==2.0.1.0",
-                "sudo /bin/sh -c 'echo {{user `version`}} > /srv/version.txt'"
+                "sudo /bin/sh -c 'echo {{user `branch`}}\t{{user `description`}} > /srv/version.txt'"
```

Comment:
This appears to be working on staging:

```bash
$ http https://staging.app.wikiwatershed.org/version.txt
HTTP/1.1 200 OK
Connection: keep-alive
Content-Encoding: gzip
Content-Length: 92
Content-Type: text/plain
Date: Tue, 09 Jan 2018 01:34:00 GMT
ETag: W/"5a541608-4e"
Last-Modified: Tue, 09 Jan 2018 01:08:24 GMT
Server: nginx
Vary: Accept-Encoding

origin/tt/version-number 1.21.0-131-gec9d821c4a90edca8c2d62416e05975dd5ab7eb9
```

However, it looks like the `\t` isn't making it through, which is causing the `undefined` of `gitDescribe` down below. For the backslash escape to be interpreted, I think that you need `echo -e`.

Reply:
I tried `echo -e` on my local but it doesn't seem to work:

```bash
$ /bin/sh -c 'echo -e this\tthat'
-e thistthat
```

I'm just switching to a space instead.